azure-quantum/azure/quantum/qiskit/backends/backend.py

```diff
@@ -114,9 +160,6 @@ def _get_input_params(self, options) -> Dict[str, Any]:
             if opt in input_params:
                 input_params[opt] = options.pop(opt)
 
-        input_params["count"] = shots_count
-        input_params["shots"] = shots_count
```

Comment:
removing the both options seems dangerous. Did we make sure that we always specify correct property for each of the providers?

Reply:
As far as I can say, everything seems ok, but I guess, we can bring that back to be on a safe side.